### ISSUE TITLE
Height of an Element challenge not working 

### ISSUE THREAD

**Issue by j-o-j-i** [freeCodeCamp/freeCodeCamp#15003]
Challenge [adjust-the-height-of-an-element-using-the-height-property](https://beta.freecodecamp.com/en/challenges/applied-visual-design/adjust-the-height-of-an-element-using-the-height-property) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

I keep getting this error message: "Your code should change the h4 height property to a value of 25 pixels."

As far as I can tell my code is correct?

```html

<style>
  h4 {
    height: 25px;
    text-align: center;
  }
  p {
    text-align: justify;
  }
  .links {
    margin-right: 20px;
    text-align: left;
  }
  .fullCard {
    width: 245px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 10px 5px;
    padding: 4px;
  }
  .cardContent {
    padding: 10px;
  }
</style>
<div class="fullCard">
  <div class="cardContent">
    <div class="cardText">
      <h4>Google</h4>
      <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>
    </div>
    <div class="cardLinks">
      <a href="https://en.wikipedia.org/wiki/Larry_Page" target="_blank" class="links">Larry Page</a>
      <a href="https://en.wikipedia.org/wiki/Sergey_Brin" target="_blank" class="links">Sergey Brin</a>
    </div>
  </div>
</div>

```

**Comment by jodylecompte** [comment]
Confirmed that the above code is passing successfully in latest Chrome on Ubuntu Linux, but latest Firefox is failing.

**Comment by Lishkon** [comment]
Attempting to solve the challenge and facing the issue described.
`h4 {
    text-align: center;
    height: 25px; 
  }`

**Comment by systimotic** [comment]
Thanks for looking into this! We recently fixed this (#15775), so I'm closing this issue.

Happy coding!
